doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -672,7 +676,9 @@ Sensor samples
 Trusted Firmware-M (TF-M) samples
 ---------------------------------
 
-|no_changes_yet_note|
+* Added:
+
+    * Support for the :ref:`nrf9151dk <zephyr:nrf9151dk_nrf9151>` board.
 
```

Comment:
Under Other samples section, add the following bullet point:

* :ref:`bootloader` sample:

  * Added support for the :ref:`zephyr:nrf9151dk_nrf9151` and the :ref:`nRF9161 DK <ug_nrf9161>` boards.

Reply:
Addded in https://github.com/nrfconnect/sdk-nrf/pull/15432